lib/elixir/test/elixir/kernel/expansion_test.exs

```diff
@@ -1180,6 +1180,16 @@ defmodule Kernel.ExpansionTest do
     assert_raise CompileError, ~r"unhandled operator ->", fn ->
       expand(quote do: (foo -> bar))
     end
+
+    assert_raise CompileError,
```

Comment:
We don't need to test warning message, it is not the purpose of this test, hence the `expand/1` usage is fine. 👍